projects/plugins/boost/changelog/fix-boost-typo

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fixed
+
+Boost: fix typo in Overall Score tooltip
```

Comment:
No need to prefix it with `Boost` since it's in the boost changelog directory. These entries go straight into the readme when releasing a new version.

You can prefix it with the aspect of the plugin that changed. In this case the UI.

```suggestion
UI: fix typo in Overall Score tooltip
```

Reply:
Thanks! I'm still wrapping my head around the different parts of the mono-repo.